webapi/src/main/scala/org/knora/webapi/routing/v2/ResourcesRouteV2.scala

```diff
@@ -376,7 +376,7 @@ class ResourcesRouteV2(routeData: KnoraRouteData) extends KnoraRoute(routeData)
 
     private def getResourcesPreviewTestResponse: Future[SourceCodeFileContent] = {
         for {
-            responseStr <- doTestDataRequest(Get(s"${baseApiUrl}resourcespreview/${SharedTestDataADM.AThing.iriEncoded}"))
+            responseStr <- doTestDataRequest(Get(s"$baseApiUrl/resourcespreview/${SharedTestDataADM.AThing.iriEncoded}"))
```

Comment:
you are still missing v2: http://0.0.0.0:3333/resourcespreview/http%3A%2F%2Frdfh.ch%2F0001%2Fa-thing

Reply:
Shall I fix it?

Reply:
I'm doing it now along with the other stuff you asked for.

Reply:
Ok, thank you.